module/bdev/raid/bdev_raid.c

```diff
@@ -2334,6 +2823,26 @@ raid_bdev_resize_base_bdev(struct spdk_bdev *base_bdev)
 	}
 }
 
+static void
+raid_bdev_remove_event_base_bdev(struct spdk_bdev *bdev)
```

Comment:
Is the naming better?

```suggestion
raid_bdev_event_remove_base_bdev(struct spdk_bdev *bdev)
```